app/assets/stylesheets/catarse_bootstrap/_main.scss

```diff
@@ -143,7 +144,8 @@ p {
 .section.full-height.hello-idea {
   padding-top: 108px;
   padding-bottom: 169px;
-  background-image: url('../images/bg-ideia.jpg');
+  background-image: url('../images/bg-icone-hello-6.jpg');
```

Comment:
I think that this is wrong, we need to use assets path like:
background-image: url('/assets/image_name')